training-node/app/controllers/user.js

```diff
@@ -85,7 +86,7 @@ exports.login = (request, response, next) => {
           const token = tokenManager.encode(userData.email);
           response.status(200);
           response.set(tokenManager.HEADER_NAME, token);
-          response.end();
+          response.send({ tokenTTL: config.common.session.expiration });
```

Comment:
there's no need of sending the expiration time. its internal

Reply:
Trello card says that the user must be informed

Reply:
Yeah i should read the card before